src/newgrf.cpp

```diff
@@ -80,7 +80,7 @@ static uint32 _ttdpatch_flags[8];
 GRFLoadedFeatures _loaded_newgrf_features;
 
 static const uint MAX_SPRITEGROUP = UINT8_MAX; ///< Maximum GRF-local ID for a spritegroup.
-static const uint MAX_GRF_COUNT = 128; ///< Maximum number of NewGRF files that could be loaded.
+static const uint MAX_GRF_COUNT = 255; ///< Maximum number of NewGRF files that could be loaded.
```

Comment:
The base set loads two NewGRFs, so practically you can now load only 253 NewGRFs in a game. Furthermore, it won't allow static NewGRFs to be loaded (for fonts or so) when the server has 253 or more NewGRFs set.